config/rbac/konflux-ci.dev_externaladmissionconfig_admin_role.yaml

```diff
@@ -0,0 +1,27 @@
+# This rule is not used by the project kueue-external-admission itself.
+# It is provided to allow the cluster admin to help manage permissions for users.
+#
+# Grants full permissions ('*') over konflux-ci.dev.konflux-ci.dev.
+# This role is intended for users authorized to modify roles and bindings within the cluster,
+# enabling them to delegate specific permissions to other users or groups as needed.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: kueue-external-admission
+    app.kubernetes.io/managed-by: kustomize
+  name: konflux-ci.dev-externaladmissionconfig-admin-role
+rules:
+- apiGroups:
+  - konflux-ci.dev.konflux-ci.dev
```

Comment:
The APIGroup listed here is incorrect.  From the CRD in this PR, the group should be `konflux-ci.dev`, not `konflux-ci.dev.konflux-ci.dev`.